docs/core/howto/application.rst

```diff
@@ -104,10 +104,10 @@ Given a file named ``my.py`` with the code:
 
 .. code-block:: python
 
-    from twisted.python.log import FileLogObserver
+    from twisted.logger import FileLogObserver, formatEventAsClassicLogText
```

Comment:
The appropriate API to use here is actually [`testFileLogObserver`](https://twistedmatrix.com/documents/17.1.0/api/twisted.logger.html#textFileLogObserver).  Would you mind updating this change?